db/models/articles-model.js

```diff
@@ -29,3 +29,16 @@ exports.selectArticles = () => {
       return rows;
     });
 };
+
+exports.updateArticle = (article_id, inc_votes) => {
+  return this.selectArticleById(article_id)
```

Comment:
this works, but we dont need to use selectArticleById here :) - we can have a Promise.reject after if it doesnt return anything!